src/ToolBox/SOS/NETCore/SOS.NETCore.csproj

```diff
@@ -39,8 +41,25 @@
     <Compile Include="SymbolReader.cs" />
   </ItemGroup>
 
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <TargetFrameworkIdentifier>.NETCoreApp</TargetFrameworkIdentifier>
+    <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8</PackageTargetFallback>
+  </PropertyGroup>
+
   <ItemGroup>
-    <None Include="project.json" />
+    <PackageReference Include="Microsoft.NETCore.Platforms">
+      <Version>$(RuntimeIdGraphDefinitionVersion)</Version>
+    </PackageReference>
+    <PackageReference Include="System.IO.FileSystem">
+      <Version>4.0.1</Version>
+    </PackageReference>
+    <PackageReference Include="System.Runtime.InteropServices">
+      <Version>4.1.0</Version>
+    </PackageReference>
+    <PackageReference Include="System.Reflection.Metadata">
+      <Version>1.4.1</Version>
+    </PackageReference>
```

Comment:
this could be shortened to:

```
<PackageReference Include="Microsoft.NETCore.Platforms" Version="$(RuntimeIdGraphDefinitionVersion)" />
<PackageReference Include="System.IO.FileSystem" Version="4.0.1" />
<PackageReference Include="System.Runtime.InteropServices" Version="4.1.0" />
<PackageReference Include="System.Reflection.Metadata" Version="1.4.1" />
```